conda_build/main_develop.py

```diff
@@ -42,6 +43,21 @@ def main():
                    help=("Relink compiled extension dependencies against "
                          "libraries found in current conda env. "
                          "Do not add source to conda.pth."))
+    p.add_argument(
+                   '-b', '--build_ext',
+                   action='store_true',
+                   help=("Build extensions inplace, invoking: "
+                         "python setup.py build_ext --inplace; "
+                         "add to conda.pth; relink runtime libraries to "
+                         "environment's lib/."))
+    p.add_argument(
+                   '-c', '--clean_build_ext',
```

Comment:
Why do you want to disallow `conda develop --clean pkg_path`? I think it should work. 

At any rate, argparse doesn't let you specify this kind of logic. You'd have to check it yourself.